### PR TITLE
env vars for repo url and ca server

### PR DESCRIPTION
**Pull request by solidstudio**

### CODE REVIEW

File: install [hunk 1]
```diff
@@ -30,6 +32,7 @@ fi
 ENV_CONFIRMED="true"
 declare -A DEFAULT_OPTS=(
   [INSTALLATION_TYPE]=interactive                  # (default interactive, options: interactive|noninteractive) Use CLI wizard to setup Swarmlet
+  [SWARMLET_REPO]=${SWARMLET_REPO}                 # (default swarmlet github url) The default repo to install
```
Comment (woudsma) [inline install:35]:
```suggestion
  [SWARMLET_REPO]=$SWARMLET_REPO                 # (default swarmlet github url) The default repo to install
```

File: modules/core/router/.env [hunk 2]
```diff
@@ -1,4 +1,5 @@
 DOMAIN=${2:-$ROOT_DOMAIN}
```
Comment (woudsma) [inline modules/core/router/.env:1]:
```suggestion
DOMAIN=$ROOT_DOMAIN
```

Same for the line below. We're not reading arguments anymore inside the `.env` file.

File: src/commands/dialog [hunk 3]
```diff
@@ -132,6 +132,30 @@ Access the Traefik dashboard at https://traefik.yourdomain.com for example."
   echo "$ROOT_DOMAIN"
 }
 
+section-ca-config() {
+  SECTION_TITLE="CA Config"
+  MESSAGE="\nUse prod CA?"
```
Comment (woudsma) [inline src/commands/dialog:137]:
```suggestion
  MESSAGE="\nLet's Encrypt configuration.\n\nUse production CA?"
```

**Comment by woudsma** [comment]
@solidstudio If you're OK with the suggested changes I will merge

**Comment by solidstudio** [comment]
yep all good!

**Pull request merged**